pkg/podexecutor/staticpod.go

```diff
@@ -197,5 +211,43 @@ func (s *StaticPod) ETCD(args executor.ETCDConfig) error {
 		HealthPort:  2381,
 		HealthPath:  "/health",
 		HealthProto: "HTTP",
+	}
+
+	if s.CISMode {
+		etcdUser, err := user.Lookup("etcd")
```

Comment:
just a question here, should we consider creating the etcd user if it doesnt exist, or at least in the install script?

Reply:
oh I see now that this mode has been added to the install script, nvm